ansible/templates/omero-ms-imageregion-config.yml.j2

```diff
@@ -0,0 +1,71 @@
+# https://github.com/glencoesoftware/omero-ms-image-region/blob/v0.5.2/src/dist/conf/config.yaml
```

Comment:
Latest release is `v0.8.5` - see https://github.com/glencoesoftware/omero-ms-image-region/blame/v0.8.5/src/dist/conf/config.yaml
But I'm not sure where the version of `omero-ms-image-region` is specified?